Collector/ApiCollector.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace SensioLabs\Bundle\ConnectBundle\Collector;
+namespace SymfonyCorp\Bundle\ConnectBundle\Collector;
 
 use Buzz\Listener\ListenerInterface;
```

Comment:
how do we decouple from this namespace so that this is compatible with symfonycorp/connect 5.1?

Reply:
I'm merging to help move forward, but could you please have a look at this @azjezz please?

Reply:
sure :+1:

Reply:
please submit against https://github.com/symfonycorp/connect-bundle